docs/ansible/roles/debops.resources/defaults-detailed.rst

```diff
@@ -305,3 +320,79 @@ Create a custom :program:`cron` task that restarts a service daily:
          #!/bin/sh
          # {{ ansible_managed }}
          test -x /usr/bin/service && systemctl restart service
+
+.. _resources__ref_acl:
+
+ACL support
+-----------
+
+Some of :ref:`debops.resources` variables also have the possibility to manage
+the ACLs (:ref:`resources__ref_paths`, :ref:`resources__ref_repositories`,
+:ref:`resources__ref_urls`, :ref:`resources__ref_archives` and
+:ref:`resources__ref_files`).
+
+Examples
+~~~~~~~~
+
+Create a directory on all hosts and allow ``adm`` group and ``joe`` user
+to access to any new content:
+
+.. code-block:: yaml
+
+   resources__paths:
+     - '/tmp/dir1'
```

Comment:
This needs to be a valid YAML dictionary key - `dest`, for example. Otherwise the YAML parser will crash.

Reply:
The example give complete RO access on the directory only for `adm` group now with two ACLs :)